### DISCUSSION TITLE
BSEED TS0011

### DISCUSSION THREAD

**Discussion by richardcc** [Koenkk/zigbee2mqtt#20292] [Devices]
I have managed to pair my BSEED TS0011touch light switches, they appear as _TZ3000_hafsqare. But they do not work correctly, their status appears as ? . However, other BSEED devices, the TS0012, do work.

Thanks

**Comment by MephistoJB** [comment]
There is already an issue open for that #20365 .

Maybe you can share the converter definition for the TS0012 so we can find a solution together.
